lib/modules.nix

```
@@ -459,7 +459,7 @@ let
         else config;
     in
     if m ? config || m ? options then
-      let badAttrs = removeAttrs m ["_class" "_file" "key" "disabledModules" "imports" "options" "config" "meta" "freeformType"]; in
+      let badAttrs = removeAttrs m ["_class" "_file" "_loc" "key" "disabledModules" "imports" "options" "config" "meta" "freeformType"]; in
```

```suggestion
      let badAttrs = removeAttrs m ["_class" "_file" "key" "disabledModules" "imports" "options" "config" "meta" "freeformType"]; in
```

This shouldn't be necessary since it's only an internal attribute